dart/lib/src/enricher/_io_enricher.dart

```diff
@@ -0,0 +1,82 @@
+import 'dart:async';
+import 'dart:io';
+
+import '../protocol.dart';
+
+import 'enricher.dart';
+
+final Enricher instance = IoEnricher();
```

Comment:
would be possible to do this with event processors instead? that's the way to mutate/enrich events

Reply:
I guess. Do we need to make sure that the information is added before or after the user's event processors?

Reply:
event processors are executed in order, our event processors should be always before the user's event processors, because when it comes to the user's event processors, every field should be already set so the user can erase/mutate the values.